src/components/PlaceOrderMethods.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div>
+    <div class="title bg-primary">place order methods</div>
+    <div class="row q-ma-md text-primary text-weight-bold">
+      <q-radio v-model="orderMethod" class="col-12 q-mb-sm" val="delivery" label="COD Delivery" @input="checkOrderMethod"/>
+      <div class="q-ml-lg text-weight-regular">
+        <q-field icon="home">
+          <q-input
+            :disable="orderMethod!=='delivery'"
+            v-model="placeOrderMothod.deliveryAddress"
+            type="textarea"
+            class="max-width"
+            placeholder="Delivery Address"
+            hide-underline
+          />
+        </q-field>
+        <q-field icon="contact_phone">
+          <q-input
+            :disable="orderMethod!=='delivery'"
+            v-model="placeOrderMothod.deliveryContact"
+            type="textarea"
+            class="max-width"
+            placeholder="Name - Phone"
+            hide-underline
+          />
+        </q-field>
+        <q-field icon="access_time">
+          <q-datetime :disable="orderMethod!=='delivery'" v-model="placeOrderMothod.deliveryTime" type="time" hide-underline/>
+        </q-field>
+      </div>
+    </div>
+    <div class="row q-ma-md text-primary text-weight-bold">
+      <q-radio v-model="orderMethod" class="col-12 q-mb-sm" val="store" label="Store Pick-up" @input="checkOrderMethod"/>
+      <div class="q-ml-lg text-weight-regular">
+        <q-field icon="store">
+          <q-select
+            v-model="placeOrderMothod.pickUpStoreId"
+            :disable="orderMethod!=='store'"
+            placeholder="Please select store"
+            :options="getRecs.map(opt=>({label:opt.name,value:opt.id}))"
+            hide-underline
+            class="max-width"
+          />
+        </q-field>
+        <q-field icon="access_time">
+          <q-datetime :disable="orderMethod!=='store'" v-model="placeOrderMothod.pickUpTime" type="time" hide-underline/>
+        </q-field>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import _d from 'lodash'
+import {mapActions, mapGetters, mapMutations} from 'vuex'
+export default {
+  props: {
+    rawData: {
+      type: Object,
+      default: () => {},
+    },
+  },
+  data() {
+    return {
+      orderMethod: 'delivery',
```

Comment:
already have isStorePickUp, why need orderMethod hardcode to 'delivery'/'store' for checking ?